src/definitions.js

```diff
@@ -29,6 +29,10 @@ export const SCRAPERS = {
     name: 'Leumi Card',
     loginFields: ['username', PASSWORD_FIELD],
   },
+  max: {
```

Comment:
@adiohana we should have a todo above leumi card to remove it in a major version

Reply:
Now I have an annoying problem.
I present a list of scrapers, which is based on `SCRAPERS`. The user enters and saves details of the scraper, along with his key (`leumiCard`) so that in the next use he will use the key to get an instance of the scraper.

So to be ready for change, I have a migration to change the key from `leumiCard` to `max`.
But it won't help me, because as long as `leumiCard` still exists in `SCRAPERS`, the user will continue to create new `leumiCard` keys.